.github/workflows/tests.yml

```diff
@@ -9,9 +9,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        #os: [ubuntu-latest, macOS-latest, windows-latest]
-        os: [ubuntu-latest, macOS-latest]
-        #java: [ '8', '11', '17' ]
+        os: [ubuntu-20.04, macOS-latest]
```

Comment:
ubuntu-latest has a newer gcc and changes some rules about alignment of `struct`s.  For now lets downgrade this.  Perhaps with Core 19, `dashj-bls` will build without issues.